### ISSUE TITLE
Photo upload : check functionality on Safari

### ISSUE THREAD

**Issue by Blandine-AA** [hhaccessibility/hhaccessibility.github.io#312]
While testing #267 , it was found that the animation to show the photo upload wasn't working. this issue is to track that.

**Comment by joshi1983** [comment]
@singlaneha2205 this looks like a testing issue to verify if there's a bug using the profile photo upload feature in Safari.  Can you try to reproduce?  I'm not sure if this was tested in an iphone or Mac.

**Comment by singlaneha2205** [comment]
@joshi1983 I tested this on Safari on iPhone and was able to upload the photo successfully and also if I try to upload a photo of size greater than 5 MB I get the appropriate error. 

There is a just a query I want to clear out here, Is the photo upload or remove photo linked to Save profile button? I mean when the user changes or uploads the photo should the Save profile button gets enabled?

**Comment by joshi1983** [comment]
No, uploading a photo saves immediately so uploading a photo should not enable the "Save Profile" button.

In fact, if you change things on the profile and upload a photo, you should actually see a dialog prompting you to save before navigating to another page.

**Comment by singlaneha2205** [comment]
@joshi1983 as suggested by you above. I changed the profile information and then tried to upload the photo without saving the changes. PFB the screenshot. Is the message displayed the expected one?
![untitled](https://user-images.githubusercontent.com/40703592/42646682-9aa84376-85cf-11e8-98b6-04e4126db33e.png)

**Comment by joshi1983** [comment]
@singlaneha2205 yes that's the expected message.  That is expected based on how it technically works.

Do you think there's a better way to handle that?

**Comment by singlaneha2205** [comment]
@joshi1983 According to my view, I think it would be more appropriate to display an error message conveying user to save profile changes first, so that user can first save the changes and then upload the image. As per now if the user selects the popup displayed the changes made by him are lost. It is my suggestion the rest can be discussed among the team to see if this is best suitable for the application.

**Comment by M4ttoF** [comment]
@singlaneha2205 I think the way it is right now is best. The current popup gives the user a choice to go back and save changes or lose the changes. If the user was messing around with settings or just didnt want to save the changes, they have the option to go back to a state they were previously comfortable with. We shouldnt be forcing them to save changes if they dont want to

**Comment by singlaneha2205** [comment]
@M4ttoF and @joshi1983 I have tested the issue on Safari browser on iPhone7 and it is working fine. But I don't have a resource to check this on MAC. If the testing on iPhone browser seems enough then we can go ahead and close the issue.
